lapce-data/src/data.rs

```diff
@@ -156,20 +167,44 @@ impl LapceData {
     }
 }
 
+/// `LapceWindowData` is the application model for a top-level window,
+/// that is a window which can be independently moved around and
+/// resized using your window manager. Normally Lapce has only one
+/// top-level window, but new ones can be created using the "New Window"
+/// command.
+/// 
+/// Each window has its own collection of tabs (again, there is normally
+/// only one tab), size, position etc. and `Arc` references to state that is
+/// common to this instance of Lapce, such as configuration and the
+/// keymap setup.
 #[derive(Clone)]
 pub struct LapceWindowData {
+    /// The unique identifier for the Window. Assigned by Druid.
     pub window_id: WindowId,
+    /// The set of tabs within the window. These tabs are high-level
+    /// groupings, in particular they are not **editor tabs**, which are
+    /// lower down the hierarchy. 
```

Comment:
For reference, would you think it made sense to link to the editor tab data struct?

Reply:
Done.